src/test/java/space/obminyashka/items_exchange/service/UserServiceIntegrationTest.java

```diff
@@ -65,13 +61,10 @@ void setUp() {
     }
 
     @Test
-    void testUpdateUserPassword_WhenDataCorrect_Successfully() {
-        UserChangePasswordDto userChangePasswordDto = new UserChangePasswordDto(CORRECT_OLD_PASSWORD, NEW_PASSWORD, NEW_PASSWORD);
-        String message = userService.updateUserPassword(userChangePasswordDto, userWithOldPassword);
-
-        assertEquals(getMessageSource(ResponseMessagesHandler.PositiveMessage.CHANGED_USER_PASSWORD), message);
-        assertTrue(bCryptPasswordEncoder.matches(NEW_PASSWORD, userWithOldPassword.getPassword()));
-        verify(userRepository).saveAndFlush(userWithOldPassword);
+    void testIsUserPasswordMatches_WhenDataCorrect_Successfully() {
+        userService.isUserPasswordMatches(userWithOldPassword.getUsername(), CORRECT_OLD_PASSWORD);
+        assertTrue(bCryptPasswordEncoder.matches(CORRECT_OLD_PASSWORD, userWithOldPassword.getPassword()));
```

Comment:
Would you mind leaving a blank like between test method and asserts part? That would allow to better find which method is a tested one.

Reply:
added